sonar-orchestrator/src/main/java/com/sonar/orchestrator/server/PackagingResolver.java

```diff
@@ -104,14 +120,16 @@ private static MavenLocation newMavenLocationOfZip(String groupId, String artifa
       .build();
   }
 
-  private Version resolveVersion(Optional<String> versionOrAlias) {
+  private Version resolveVersion(SonarDistribution sonarDistribution) {
+    Optional<String> versionOrAlias = sonarDistribution.getVersion();
     if (!versionOrAlias.isPresent()) {
       throw new IllegalStateException("Missing SonarQube version");
     }
 
     String version = versionOrAlias.get();
     if (version.startsWith("DEV") || version.startsWith("LATEST_RELEASE") || version.startsWith("DOGFOOD")) {
-      MavenLocation location = newMavenLocationOfZip(PUBLIC_GROUP_ID, "sonar-application", version);
+      MavenCoordinates mavenCoordinates = getMavenCoordinates(sonarDistribution, Version.create("7.2"));
```

Comment:
7.2 is the first version the results in the expected behavior.
It can be probably encapsulated better or we can think about dropping specific use cases for SQ 7.2.